cmd/block-generator/generator/generate.go

```diff
@@ -616,29 +589,28 @@ func (g *generator) generateAssetTxnInternalHint(txType TxTypeID, sp sdk_types.S
 		os.Exit(1)
 	}
 
-	txn.Fee = 1000
-	if g.balances[senderIndex] < uint64(txn.Fee) {
+	if g.balances[senderIndex] < fee {
 		fmt.Printf(fmt.Sprintf("\n\nthe sender account does not have enough algos for the transfer. idx %d, asset transaction type %v, num %d\n\n", senderIndex, actual, g.reportData[actual].GenerationCount))
 		os.Exit(1)
 	}
-	g.balances[senderIndex] -= uint64(txn.Fee)
+	g.balances[senderIndex] -= fee
```

Comment:
Please revert these fee changes. I don't mind hard coding it in the make functions, but using the hard coded value here requires knowledge of implementation details.

Reply:
makes sense